Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.cpp

```diff
@@ -813,11 +812,12 @@ namespace Terrain
             return;
         }
 
-        const float rcpWorldZ = 1.0f / m_worldBounds.GetExtents().GetZ();
+        float zExtents = (m_worldMaxHeight - m_worldMinHeight);
+        const float rcpWorldZ = 1.0f / (m_worldMaxHeight - m_worldMinHeight);
```

Comment:
Can be `1.0f / zExtents`

Reply:
Nice catch - I'll change it for this review if AR fails, but it's minor enough where I'll just add it to my next PR if AR succeeds